plugin-build/src/test/kotlin/io/sentry/android/gradle/integration/SentryPluginTest.kt

```diff
@@ -246,9 +246,8 @@ class SentryPluginTest :
             .appendArguments(":app:assembleDebug", "--info")
             .build()
 
-        val instr = "OkHttpEventListener, OkHttp"
         assertTrue {
-            "[sentry] Instrumentable: ChainedInstrumentable(instrumentables=$instr)" in build.output
+            "[sentry] Instrumentable: ChainedInstrumentable(instrumentables=OkHttp)" in build.output
```

Comment:
I have no clue how this has been passing before 🤷 @adinauer you said once, it was failing for you because you introduced this change, but I think the test is supposed to actually test that only `OkHttp` instrumentation is applied, without the listener

Reply:
Ok, figured, it was the other test for jvm which was overwriting the build files, so the test was failing depending on the order of execution. I've removed it altogether, since we anyway test that in `SentryPluginNonAndroidAutoInstallTest`